coding-concepts.js

```diff
@@ -8,36 +8,36 @@
 // --------------------1) What will this log?
 
 const cohort = "Golf 2022"
-// console.log(cohort.split(""))
+console.log(cohort.split(""))
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: ["G", "o", "l", "f", " ", "2", "0", "2", "2"]
+// b) Verify and explain: Correct! .split built in method transforms the string into an array with each character in its own index, including the space.
 
 // --------------------2) What will this log?
 
 const greeter = (name) => {
   `Hello, ${name}!`
 }
-// console.log(greeter("LEARN Student"))
+console.log(greeter("LEARN Student"))
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: Hello, LEARN Student
+// b) Verify and explain: undefined. I seem to have overlooked the lack of a return in this function.
```

Comment:
For sure, this can be easy to overlook.  Attention to small details is super important in development.